test/actions.js

```diff
@@ -32,14 +32,14 @@ const { getSecondTokenAmount } = require('./utils-uni');
 const { LiquityActionIds, getHints, getRedemptionHints } = require('./utils-liquity');
 const { execShellCommand } = require('../scripts/hardhat-tasks-functions');
 
-const executeAction = async (actionName, functionData, proxy, regAddr = REGISTRY_ADDR) => {
```

Comment:
removing regAddr = REGISTRY_ADDR breaks forkoor related stuff which need to inject registry like mcdWithdraw action